mobile/src/actions/index.js

```diff
@@ -92,6 +92,15 @@ const onRegistered = dispatch => (client, url) => {
         throw err
       }
     )
+  } else {
+    // TODO open a new tab ?
```

Comment:
As there is no inapp browser, we need to open the `oauth/` page in a new tab (or maybe an iframe) then get the response's url to give it back to `cozy-client-js`. For now, a prompt ask for this url (the one with `access_code` query parameter) and it initialize `cozy-client-js` letting it to add the token in its persistence layer.